packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -224,6 +225,10 @@ class GestureDetector extends StatelessWidget {
   /// a long period of time.
   final GestureLongPressCallback onLongPress;
 
+  /// A pointer has left the contact with the screen at the same location after
+  /// a long period of time.
+  final GestureLongPressUpCallback onLongPressUp;
```

Comment:
maybe "A pointer that has triggered a long-press has stopped contacting the screen."